Packs/FeedRSS/Integrations/FeedRSS/FeedRSS.py

```diff
@@ -0,0 +1,185 @@
+            return ""
+        if len(encoded_content) > self.content_max_size:  # Ensure report_content does not exceed the
+            # indicator size limit (~50KB)
+            report_content = encoded_content[:self.content_max_size].decode('utf-8')
+            report_content += ' This is truncated text, report content was too big.'
+
+        return report_content
+
+
+def fetch_indicators(client: Client):
+    client.parse_feed_data()
+    parsed_indicators = client.create_indicators_from_response()
+    return parsed_indicators
+
+
+def get_indicators(client: Client, indicators: list, args: dict) -> CommandResults:
+    limit = int(args.get('limit', 10))
+    parsed_indicators = indicators[:limit]
+    parsed_for_hr = []
+    for indicator in parsed_indicators:
+        link = indicator.get('fields', {}).get('publications', [{}])[0].get('link')
+        article_title = indicator.get('value', '')
+        article_field_hr = article_title
+        if link:
+            article_field_hr = f"[{article_title}]({link})"  # if there is a link to the article, we want the
+            # article's title to be a link.
+        parsed_for_hr.append({'Article': article_field_hr,
+                              'Type': indicator.get('type')})
+    headers = ['Article', 'Type']
+    hr_ = tableToMarkdown(name='RSS Feed:', t=parsed_for_hr, headers=headers)
+    return CommandResults(
+        readable_output=hr_,
+        raw_response=client.feed_data
+    )
+
+
+def check_feed(client: Client):
+    if client.feed_response and 'html' in client.feed_response.headers['content-type']:
+        raise DemistoException(f'{client._base_url} is not rss feed url. Try look for a url containing xml format data,'
+                               f' that could be found under urls with \'feed\' prefix or suffix.')
+    else:
+        return_results("ok")
+
+
+def main():
+    params = demisto.params()
+    server_url = params.get('server_url')
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+
+    try:
+        reliability = params.get('feedReliability')
+        reliability = reliability if reliability else DBotScoreReliability.F
+
+        if DBotScoreReliability.is_valid_type(reliability):
+            reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+        else:
+            raise Exception("Please provide a valid value for the Source Reliability parameter.")
+        client = Client(server_url=server_url,
+                        use_ssl=not params.get('insecure', False),
+                        proxy=params.get('proxy'),
+                        reliability=reliability,
+                        feed_tags=argToList(params.get('feedTags')),
+                        tlp_color=params.get('tlp_color'),
+                        content_max_size=int(params.get('max_size', '45')))
+
+        client.request_feed_url()
```

Comment:
I suggest requesting the feed url in each command(test/fetch/get) and return the result, the feed response is not a client property.